randovania/games/prime1/logic_database/Impact Crater.txt

```diff
@@ -247,12 +247,8 @@ Extra - aabb: [-53.122448, -399.23877, -44.54401, 2.4719353, -265.25775, 6.13683
           Charge Beam or Combat (Intermediate)
           Any of the following:
               Shoot Power Beam
-              All of the following:
-                  # Fight Without Power Beam
-                  Charge Beam and Flamethrower and Plasma Beam and Combat (Intermediate) and Knowledge (Intermediate)
-                  Any of the following:
-                      Missile ≥ 50
-                      Missile ≥ 25 and Disabled Hard Mode
+              # Fight Without Power Beam
+              Charge Beam and Flamethrower and Missile ≥ 25 and Combat (Intermediate) and Knowledge (Intermediate) and Disabled Hard Mode and Shoot Plasma Beam
```

Comment:
The missile requirements here got changed entirely. Can you kill exo in this phase with flamethrower really only on normal/easy mode?

Reply:
I accidentally removed the hard mode requirements. Will fix.